R/modeltests.R

```diff
@@ -140,15 +139,26 @@ startRuns <- function(test, model, mydir, gitPath, user) {
                     "startgroup=AMT titletag=AMT slurmConfig=\"--qos=standby --nodes=1 --tasks-per-node=12\" ",
                     "config/scenario_config.csv"))
 
+      # Create and save a list of runs that should have been started in order to determine later which runs were not started
       settings <- read.csv2("config/scenario_config.csv",
                              stringsAsFactors = FALSE,
                              row.names = 1,
                              comment.char = "#",
                              na.strings = "")
-
       runsToStart <- selectScenarios(settings = settings, interactive = FALSE, startgroup = "AMT")
       row.names(runsToStart) <- paste0(row.names(runsToStart), "-AMT")
       saveRDS(runsToStart, file = paste0(mydir, "/runsToStart.rds"))
+      
+      # start make test-full
+      # 1. pull changes to magpie develop
+      withr::with_dir("magpie", {
+            system(paste0(gitPath, " reset --hard origin/develop && ", gitPath, " pull"))
```

Comment:
```suggestion
            system(paste0(gitPath, " reset --hard origin/master && ", gitPath, " pull"))
```
Do we really want to test against MAgPIE develop? In case there is some temporaty error in MAgPIE, why should we have REMIND fails? Not sure about that.

Reply:
I agree, it is a bit of a risk. However, the advantage is that it allows us to detect changes in the coupling interface immediately (= when the name of variables change that are required for the coupling) and not after it has already been established.

Reply:
I agree, this is actually the better choice.